### ISSUE TITLE
Microsoft.Storage/storageAccounts/blobServices CorsRules not in Pulumi.AzureNextGen.Storage.Latest

### ISSUE THREAD

**Issue by jamesianberry** [pulumi/pulumi-azure-nextgen#91]
#78 Had to revert to existing Azure provider to provision storage account with Cors rules for Blob services:
```
using Azure = Pulumi.Azure;
using Pulumi.AzureNextGen.Resources.Latest;
using Storage = Pulumi.AzureNextGen.Storage.Latest;
...
        var resourceGroupName = args.ResourceGroup.Name;
//        var egDataStorage = new Storage.StorageAccount("EgDataStorage", new Storage.StorageAccountArgs
//        {
//            AccessTier = "Hot",
//            AccountName = args.StorageAccountName,
//            EnableHttpsTrafficOnly = false,
//            Encryption = new Storage.Inputs.EncryptionArgs
//            {
//                KeySource = "Microsoft.Storage",
//                Services = new Storage.Inputs.EncryptionServicesArgs
//                {
//                    Blob = new Storage.Inputs.EncryptionServiceArgs
//                    {
//                        Enabled = true,
//                        KeyType = "Account",
//                    },
//                    File = new Storage.Inputs.EncryptionServiceArgs
//                    {
//                        Enabled = true,
//                        KeyType = "Account",
//                    },
//                },
//            },
//            Kind = "StorageV2",
//            Location = args.ResourceGroup.Location,
//            NetworkRuleSet = new Storage.Inputs.NetworkRuleSetArgs
//            {
//                Bypass = "AzureServices",
//                DefaultAction = "Allow",
//                IpRules = {},
//                VirtualNetworkRules = {},
//            },
//            ResourceGroupName = resourceGroupName,
//            Sku = new Storage.Inputs.SkuArgs
//            {
//                Name = "Standard_LRS",
//                //Tier = "Standard",
//            },
//        });
        var egDataStorage = new Azure.Storage.Account("EgDataStorage", new Azure.Storage.AccountArgs
        {
            AccessTier = "Hot",
            AccountKind = "StorageV2",
            AccountReplicationType = "LRS",
            AccountTier = "Standard",
            AllowBlobPublicAccess = true,
            BlobProperties = new Azure.Storage.Inputs.AccountBlobPropertiesArgs
            {
                CorsRules = new Azure.Storage.Inputs.AccountBlobPropertiesCorsRuleArgs
                {
                    AllowedHeaders = {"*"},
                    AllowedMethods = {"GET", "PUT"},
                    AllowedOrigins = {"*"},
                    ExposedHeaders = {"*"},
                    MaxAgeInSeconds = 1000,
                },
            },
            EnableHttpsTrafficOnly = false,
            Location = args.ResourceGroup.Location,
            Name = args.StorageAccountName,
            NetworkRules = new Azure.Storage.Inputs.AccountNetworkRulesArgs
            {
                Bypasses = {"AzureServices"},
                DefaultAction = "Allow",
                IpRules = {},
                VirtualNetworkSubnetIds = {},
            },
            ResourceGroupName = resourceGroupName,
        });
```
Works fine using both providers so no rush...

**Comment by justinvp** [comment]
Dupe of https://github.com/pulumi/pulumi-azure-nextgen/issues/13?

**Comment by mikhailshilkov** [comment]
@justinvp No, it's different, I'll take a look

**Comment by jamesianberry** [comment]
Confirmed fixed in 0.2.5 with this example code:
```
        var egBlobServiceProperties = new Storage.BlobServiceProperties("EgBlobServicesProperties", new Storage.BlobServicePropertiesArgs
        {
            AccountName = egStorage.Name,
            BlobServicesName = "default",
            Cors = new Storage.Inputs.CorsRulesArgs
            {
                CorsRules = new Storage.Inputs.CorsRuleArgs
                {
                    AllowedHeaders = {"*"},
                    AllowedMethods = {"GET", "PUT"},
                    AllowedOrigins = {"*"},
                    ExposedHeaders = {"*"},
                    MaxAgeInSeconds = 1000,
                },
            },
            ResourceGroupName = args.ResourceGroup.Name,
        });
```
